db/install.xml

```diff
@@ -54,6 +54,10 @@
         <FIELD NAME="fingerprint" TYPE="char" LENGTH="255" NOTNULL="false" SEQUENCE="false"/>
         <FIELD NAME="cool" TYPE="int" LENGTH="10" NOTNULL="false" DEFAULT="0" SEQUENCE="false"/>
         <FIELD NAME="cooldate" TYPE="int" LENGTH="10" NOTNULL="false" DEFAULT="0" SEQUENCE="false"/>
+        <FIELD NAME="mobileservicesenabled" TYPE="int" LENGTH="1" NOTNULL="false" DEFAULT="-1" SEQUENCE="false" COMMENT="Whether mobile services are enabled in the site, -1 if unknown"/>
+        <FIELD NAME="mobilenotificacionsenabled" TYPE="int" LENGTH="1" NOTNULL="false" DEFAULT="-1" SEQUENCE="false" COMMENT="Whether mobile notifications are enabled in the site, -1 if unknown"/>
```

Comment:
Hi @jleyva,
typo in the field name: it looks like a bit of spanglish here :smile: .

HTH,
Matteo